src/styles/vars.scss

```diff
@@ -0,0 +1,7 @@
+$primary-bg-color: #f3f3f3;
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-main-color: #00acdc;
+$background-card-color: #fff;
+$hover-color: #34568b;
```

Comment:
I see you did not use this variable anywhere in the code.
However, you should use it accordingly to [this part of the Checklist](https://github.com/mate-academy/layout_catalog#:~:text=change%20the%20card%20title%20text%20color%20to%20%2334568b%20when%20the%20card%20is%20hovered%20(.card%3Ahover%20.card__title))